en/services/payload_control.md

```diff
@@ -0,0 +1,62 @@
+
+
+Message | Description
+-- | --
+<a id="MAV_CMD_DO_GRIPPER"></a>[MAV_CMD_DO_GRIPPER](../messages/common.md#MAV_CMD_DO_GRIPPER) | Command to engage and release a gripper.
+<a id="MAV_CMD_DO_WINCH"></a>[MAV_CMD_DO_WINCH](../messages/common.md#MAV_CMD_DO_WINCH) | Command to operate a specified winch.
+<a id="MAV_CMD_ILLUMINATOR_ON_OFF"></a>[MAV_CMD_ILLUMINATOR_ON_OFF](../messages/common.md#MAV_CMD_ILLUMINATOR_ON_OFF) | Command to turn illuminators ON/OFF. An illuminator is a light source that for lighting up dark areas external to the sytstem, such as a torch or searchlight.
+<a id="MAV_CMD_DO_SET_CAM_TRIGG_INTERVAL"></a>[MAV_CMD_DO_SET_CAM_TRIGG_INTERVAL](../messages/common.md#MAV_CMD_DO_SET_CAM_TRIGG_INTERVAL) | Command to set time interval between camera captures
+<a id="MAV_CMD_DO_SET_CAM_TRIGG_DIST"></a>[MAV_CMD_DO_SET_CAM_TRIGG_DIST](../messages/common.md#MAV_CMD_DO_SET_CAM_TRIGG_DIST) | Command to set distance travelled between camera captures.
+
+
+## Arbitrary/Unknown Payload Commands
+
+The [MAV_CMD_DO_SET_ACTUATOR](#MAV_CMD_DO_SET_ACTUATOR) and [MAV_CMD_DO_SET_SERVO](#MAV_CMD_DO_SET_SERVO) commands allow setting the value of flight controller outputs, without having to know what that specific hardware is. 
+
+> **Tip** Use the [commands for known payload types](#payload-specific-commands) where possible.
+  These are more intuitive for users, and in logs.
+
+`MAV_CMD_DO_SET_ACTUATOR` is better designed and more recent than `MAV_CMD_DO_SET_SERVO`, but has not yet been widely adopted.
+- `DO_SET_ACTUATOR` can in theory be used on any bus - MAIN, AUX, UAVCAN, etc., while `DO_SET_SERVO` can only be used on PWM busses, and is generally hardcoded to map instance numbers to AUX outputs (this is not mandated by the specification, so cannot be relied on).
+- `DO_SET_ACTUATOR` can set multiple outputs in a single message. 
+
+Message | Description
+-- | --
+<a id="MAV_CMD_DO_SET_ACTUATOR"></a>[MAV_CMD_DO_SET_ACTUATOR](../messages/common.md#MAV_CMD_DO_SET_ACTUATOR) | Sets actuators (e.g. servos) to a desired value. The actuator numbers are mapped to specific outputs (e.g. on any MAIN or AUX PWM or UAVCAN) using a flight-stack specific mechanism (for example, parameters).
+<a id="MAV_CMD_DO_SET_SERVO"></a>[MAV_CMD_DO_SET_SERVO](../messages/common.md#MAV_CMD_DO_SET_SERVO) | Sets a servo, identified by a specified instance number, to a specified PWM value.
+
+
+> **Note** **Implementations:**
+  - `MAV_CMD_DO_SET_SERVO` is impemented on both ArduPilot and PX4.
+    In both cases instance numbers map to AUX outputs.
+  - `MAV_CMD_DO_SET_ACTUATOR` is impemented only on PX4 (June 2021).
+    Only three outputs can be set (param 1, 2, 3), and these are mapped by parameters to AUX outputs chosen by the user (though there is nothing to stop them being mapped to other busses).
+	For more information see the [PX4 User Guide](https://docs.px4.io/master/en/payloads/#mission-triggering).
+
+## Payload Deployment Commands
+
```

Comment:
@bkueng @julianoes @LorenzMeier @auturgy  Do any of you have any idea about these commands, or who might? 

They are kind of orthogonal to the other commands and look like they are designed to control dumping a set of payloads in a mission. However there is not enough information to infer the usage and there is some overlap. In addition, it is not clear what "payload" means - the implication is that maybe you're dropping off multiple things using a gripper or whatever.

Reply:
Illuminator was added by a PX4 user from memory: whether the implementation was pushed upstream I have no idea. Gripper, winch (and Sprayer) are used in ArduPilot.

Reply:
Thanks. I'll chase up the status of illuminator. My bigger concern is the MAV_CMD_PAYLOAD_ variants - because it is not clear how they would be used. Have added question to dev call https://github.com/mavlink/mavlink/wiki/20210609-Dev-Meeting